src/status_im2/contexts/onboarding/events.cljs

```diff
@@ -35,11 +35,30 @@
   {:biometric/authenticate {:on-success #(rf/dispatch [:onboarding-2/biometrics-done])
                             :on-fail    #(rf/dispatch [:onboarding-2/biometrics-fail %])}})
 
+(rf/defn authenticate-enable-biometrics
```

Comment:
@jo-mut - I think there is a slightly better approach to take here. I will make a suggested commit and you can let me know what you think 👍

Reply:
please see 1392a4eee015f4d3173ffa9134f765c04ffd8099 - I have this code pushed on branch `jc/standard-auth-refactor`

The problem here is that the standard auth code only exposes a slide button, so you had to make these adjustments for it to work with the quo/button. 

The better approach imo is to add another component in standard-auth "module" - which uses quo/button component under the hood, we can then encapsulate all of this authorization logic in there 👍 
I think we will need  for many other screens anyway so it would be good to have this added in 👍

Reply:
I made a slight adjustment - 1ad86267d9306fba9ad5d26087c40c13531d04be (also updated requires in wallet screens 👍 )

Reply:
@J-Son89 I have looked at your branch. I agree with your suggestions. thanks